tango/test_context.py

```diff
@@ -26,9 +26,10 @@
 
 # Local imports
 from .server import run
+from .utils import is_non_str_seq
 from . import DeviceProxy, Database, Util
 
-__all__ = ("DeviceTestContext", "run_device_test_context")
+__all__ = ("MultiDeviceTestContext", "DeviceTestContext", "run_device_test_context")
```

Comment:
I just noticed that this is also done in the test_utils module, so please add something similar there:
https://github.com/tango-controls/pytango/blob/492c8752650744b88112bebc450f8497a8b0315b/tango/test_utils.py#L27

Reply:
Done in db5de328